src/main/java/org/dbos/apiary/utilities/Percentile.java

```diff
@@ -0,0 +1,46 @@
+package org.dbos.apiary.utilities;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+public class Percentile {
+    private List<Long> times = new ArrayList<>();
```

Comment:
This `times` is not thread-safe. Could switch to use `ConcurrentLinkedQueue` instead.